inference-engine/tests/functional/shared_test_classes/include/shared_test_classes/single_layer/softmax.hpp

```diff
@@ -17,15 +17,16 @@
 namespace LayerTestsDefinitions {
 
 using softMaxLayerTestParams = std::tuple<
-        InferenceEngine::Precision,         // netPrecision
-        InferenceEngine::Precision,         // Input precision
-        InferenceEngine::Precision,         // Output precision
-        InferenceEngine::Layout,            // Input layout
-        InferenceEngine::Layout,            // Output layout
-        InferenceEngine::SizeVector,        // inputShape
-        size_t,                             // axis
-        std::string,                        // targetDevice
-        std::map<std::string, std::string>  // config
+        InferenceEngine::Precision,                          // netPrecision
+        InferenceEngine::Precision,                          // Input precision
+        InferenceEngine::Precision,                          // Output precision
+        InferenceEngine::Layout,                             // Input layout
+        InferenceEngine::Layout,                             // Output layout
+        std::vector<std::vector<std::pair<size_t, size_t>>>, // Input shape
```

Comment:
The same commen5t as for convolution

Reply:
Yes, and this is planned to support eltwise style input as well.

Reply:
Yes, this approach is correct for multinput nodes, not for one input. I mean that we should not overload instantiations. but in the Layer_test_common it should contain logic for multiinput nodes

Reply:
Maxim coded to use copied inputs for one input. So I just assumed like that. And your suggestion may be better.

Reply:
It doesn't mean that we should use Maxim's approach in all test cases. It's applicable for Multiinput nodes I think

Reply:
You meant supporting eltwise style inputs only, right?